lib/Extension/LanguageServerReferenceFinder/Tests/Unit/Handler/GotoDefinitionHandlerTest.php

```diff
@@ -31,10 +29,11 @@ class GotoDefinitionHandlerTest extends TestCase
     public function testGoesToDefinition(): void
     {
         $document = TextDocumentBuilder::create(self::EXAMPLE_TEXT)->uri(self::EXAMPLE_URI)->build();
+
         $locations = [
             new TypeLocation(
                 TypeFactory::class('Foo'),
-                new DefinitionLocation($document->uri(), ByteOffset::fromInt(2))
+                new PhpactorLocation($document->uriOrThrow(), ByteOffset::fromInt(2))
```

Comment:
can we remove the alias and just have `Location` ?

Reply:
In this file there is this `Phpactor\TextDocument\Location` and `Phpactor\LanguageServerProtocol\Location` the later one is not aliased so the first one needs an alias.